src/platforms/common/blackpill-f4/Makefile.inc

```
@@ -29,9 +41,11 @@ ifdef ALTERNATIVE_PINOUT
 CFLAGS += -DALTERNATIVE_PINOUT=$(ALTERNATIVE_PINOUT)
 endif
 
-VPATH += platforms/common/stm32
+VPATH += platforms/common/stm32 \
+	platforms/common/blackpill-f4
```

Stylistically this should be split up as
```suggestion
VPATH +=                          \
    platforms/common/stm32        \
    platforms/common/blackpill-f4
```
However, this works and there's no real set style for the Makefiles at the moment, so this is an entirely optional change

---

Yes I agree, your proposal looks much neater.

In commit 43ad004 I have updated the coding style used throughout the Makefile, aligning the backslashes in each section.